testcafe/e2e/changeLanguage.ts

```diff
@@ -4,9 +4,12 @@ import translationsFi from '../../src/common/translation/i18n/fi.json';
 import translationsSv from '../../src/common/translation/i18n/sv.json';
 import { header } from '../selectors/header';
 import { getPathname } from '../utils/clientUtils';
+import jsErrorHandler from '../utils/jsErrorHandler';
 import { getEnvUrl } from '../utils/settings';
 
-fixture('Landing page').page(getEnvUrl('/fi/home'));
+fixture('Landing page')
+  .clientScripts(jsErrorHandler)
```

Comment:
Very god tha there was solution for this. Should we add small comment why this is added? In addition to the see more link

Reply:
added

Reply:
Great, for some reason I have missed this change and comment